srcpkgs/python3-Pyro4/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'python3-Pyro4'
+pkgname=python3-Pyro4
+version=4.75
+revision=1
+archs=noarch
+wrksrc="${pkgname#*-}-${version}"
+build_style=python3-module
+pycompile_module="Pyro4"
+hostmakedepends="python3-setuptools"
+depends="python3-serpent>=1.27 python3-selectors2"
+short_desc="Distributed Object Middleware (IPC/RPC) for Python2"
+maintainer="pulux <pulux@pf4sh.de>"
+license="MIT"
+homepage="https://pythonhosted.org/Pyro4/"
+distfiles="${PYPI_SITE}/P/Pyro4/Pyro4-${version}.tar.gz"
+checksum=3897c0254046d4cb412a4d1a8f2f9c2c1c1ae643a24db07d0abdb51acdb8d7b5
+
+alternatives="
+ pyro4:pyro4-check-config:/usr/bin/pyro4-check-config2
+ pyro4:pyro4-flameserver:/usr/bin/pyro4-flameserver2
+ pyro4:pyro4-httpgateway:/usr/bin/pyro4-httpgateway2
+ pyro4:pyro4-ns:/usr/bin/pyro4-ns2
+ pyro4:pyro4-nsc:/usr/bin/pyro4-nsc2
+ pyro4:pyro4-test-echoserver:/usr/bin/pyro4-test-echoserver2"
```

Comment:
Drop the alternates!
`build_style=python3-module` doesn't create versioned executable.

Even if they created versioned executable, 2 suffix indicated python2 version.

Reply:
Didn’t actually add these ;) but ok